apps/tlon-web/src/chat/ChatScroller/ChatScroller.tsx

```diff
@@ -2,6 +2,7 @@ import { Virtualizer, useVirtualizer } from '@tanstack/react-virtual';
 import { PostTuple, ReplyTuple } from '@tloncorp/shared/dist/urbit/channel';
 import { WritTuple } from '@tloncorp/shared/dist/urbit/dms';
 import { BigInteger } from 'big-integer';
+import { MessageKey } from 'packages/shared/dist/urbit/activity';
```

Comment:
```suggestion
import { MessageKey } from '@tloncorp/shared/dist/urbit/activity';
```
bad import statement